forge-game/src/main/java/forge/game/ability/effects/PlayEffect.java

```diff
@@ -363,7 +363,12 @@ public void resolve(final SpellAbility sa) {
                 if ("ManaCost".equals(sa.getParam("PlayCost"))) {
                     abCost = new Cost(source.getManaCost(), false);
                 } else {
-                    abCost = new Cost(sa.getParam("PlayCost"), false);
+                    String cost = sa.getParam("PlayCost");
```

Comment:
If you move this String assignment a bit higher you can also reuse it in the first statement